tests/scripts/environment.py

```diff
@@ -12,8 +13,13 @@ def __init__(self, **kwargs):
         self._env = kwargs
 
     def __getitem__(self, key):
-        environment_variable = self._env[key]
-        value = os.environ[environment_variable]
+        try:
+            environment_variable = self._env[key]
+            value = os.environ[environment_variable]
+        except KeyError:
+            warn(f'KeyError occured when attempting to retrieve {key} from environment variables.')
+            value = None
```

Comment:
Did not want to use `.get(key, default)` for this as knowing the env var is missing is very useful, especially if one hasn't set up the environment completely.